app/controllers/api/v0/admins_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V0
+    class AdminsController < BaseController
+      def add_portfolio
+        portfolio = Portfolio.create!(portfolio_params)
+        render json: portfolio
```

Comment:
The creates should also render `:status => :created, :location => link_to_the_created_record`